CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
     - Table:
       - CHANGED: switch to pre-calculated distances for table responses for large speedup and 10% memory increase. [#5251](https://github.com/Project-OSRM/osrm-backend/pull/5251)
       - ADDED: new parameter `fallback_speed` which will fill `null` cells with estimated value [#5257](https://github.com/Project-OSRM/osrm-backend/pull/5257)
+      - ADDED: if `fallback_speed` is used, a new structure `estimated_cells` will describe which cells contain estimated values [#5259](https://github.com/Project-OSRM/osrm-backend/pull/5259)
```

Comment:
Should the `estimated_cells` feature get a flag of it's own? In some cases this information may not be needed and would be unused extra bytes.

Reply:
Let's put that into a different PR!